creativecloud/scripts/utils.js

```diff
@@ -22,7 +22,11 @@
 export const [setLibs, getLibs] = (() => {
   let libs;
   return [
-    (prodLibs) => {
+    (prodLibs, force = false) => {
```

Comment:
i would still as discussed call this with a more explicit name on what exactly does it force :) 

also i wonder what is the point of having 
`const foo = getLibs('blah', true)` vs `const foo = 'blah'`

Reply:
there is no `const foo = getLibs('blah', true)` 
only `const foo = setLibs('blah', true)`

Reply:
force is pretty self explanatory no?